tests/observability/metrics/test_vms_metrics.py

```
@@ -576,9 +576,15 @@ def test_kubevirt_vmi_sync_total(self):
             - Two metric entries are returned — one from virt-controller
               and one from virt-handler — each with a value greater than 0
         """
+        validate_vmi_sync_total_reported_and_positive(
+            prometheus=prometheus,
+            metric_query=KUBEVIRT_VMI_SYNC_TOTAL.format(vm_name=vm_for_migration_metrics_test.name),
+        )
 
     @pytest.mark.polarion("CNV-16272")
-    def test_kubevirt_vmi_sync_total_increases_after_migration(self):
+    def test_kubevirt_vmi_sync_total_increases_after_migration(
+        self, prometheus, initial_vmi_sync_total_values, vm_for_migration_metrics_test, migration_succeeded_scope_class
```

Ruff (0.15.18)

[warning] Unused method argument: `migration_succeeded_scope_class`

(ARG002)